extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -333,6 +337,40 @@ export class Bind {
     return this.history_;
   }
 
+  /**
+   * Calls setState(s), where s is data.state with the non-overridable keys removed.
+   * @param {string} eventType
+   * @param {*} data
+   * @return {!Promise}
+   * @private
+   */
+  premutate_(eventType, data) {
```

Comment:
Are you planning to use `awaitResponse`?

Reply:
No, I didn't end up using it. I removed `eventType` and `awaitResponse` from the function signature for the handlers now.